scripts/migration/migrate_deleted_wikis.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.WIKI_DELETED))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    for log in targets:
+        node = log.node
+        versions = node.wiki_pages_versions
+        current = node.wiki_pages_current
+        updated_versions = {}
+        for wiki in versions:
+            if wiki in current:
+                updated_versions[wiki] = versions[wiki]
+        if not dry_run:
```

Comment:
Yeah, this shouldn't be necessary. Raising an uncaught exception is currently the preferred way to implement dry_runs. 

One minor refactor would be to raise the error in the with block:

```python
with TokuTransaction():
     migrate(...)
      if dry_run:
           raise RuntimeError(...)
```

This puts all the transaction code in the same place.